physics/GFS_surface_composites.F90

```diff
@@ -137,9 +161,11 @@ subroutine GFS_surface_composites_pre_run (im, flag_init, flag_restart, lkm, fra
       else
 
         do i = 1, IM
+          oceanfrac(i) = one -  landfrac(i) - lakefrac(i)
```

Comment:
In some routines they assume one=oceanfrac + landfrac, so lake is either part of ocean or land. For example, in atmos_model.F90

!$omp parallel do default(shared) private(i,j,nb,ix)
    do j=jsc,jec
      do i=isc,iec
        nb = Atm_block%blkno(i,j)
        ix = Atm_block%ixp(i,j)
! use land sea mask: land:1, ocean:0
        lsmask(i,j) = floor(one + epsln - GFS_data(nb)%SfcProp%oceanfrac(ix))
      enddo
    enddo

So we need to double-check if lakefrac+oceanfrac+landfrac=1 in the oro data,  otherwise there will be some mismatches for land-sea mask, which will cause the model crash.